src/pyscript/plugins/create.py

```diff
@@ -1,15 +1,61 @@
+import time
+import webbrowser
+from pathlib import Path
+from typing import Optional
+
 import typer
 
-from pyscript import LATEST_PYSCRIPT_VERSION, app, cli, plugins
-from pyscript._generator import create_project
+from pyscript import LATEST_PYSCRIPT_VERSION, app, cli, console, plugins
+from pyscript._generator import create_project, file_to_html, string_to_html
+
+
+def wrap_file(title, output, command, show, app_or_file_name, pyscript_version):
+    title = title or "PyScript App"
+
+    # Derive the output path if it is not provided
+    remove_output = False
+    if output is None:
+        if command and show:
+            output = Path("pyscript_tmp.html")
+            remove_output = True
```

Comment:
I don't think I understand this. Can you explain what happens here and why we need to remove the output?

Reply:
This is if we are using `pyscript create --wrap -c 'print("Hi!")' --show` let's say aka a command line string with the
`--show` option, then we create a temporary HTML file, open it in the browser (without the server), and then delete it later.

This is also borrowed / carried from what `pyscript wrap` was doing earlier.

Reply:
No longer needed since implementation has changed and `show` is also removed.